swap_meet/vendor.py

```diff
@@ -1,2 +1,92 @@
+""" Initiation of class Vendor """
+
 class Vendor:
-    pass
\ No newline at end of file
+		""" Represents a unique entity from which one can trade items. """
+		def __init__(self, inventory=None):
+				""" Initiate class vendor with empty inventory. """
+				self.inventory = [] if inventory is None else inventory
+
+		def add (self, item):
+			""" Adds an item to inventory and returns the added item """
+			self.inventory.append(item)
+			return item
+
+
+		def remove (self, item):
+				""" Remove one matching item from the 'inventory' and returns it. Returns False if no matching item. """
+				for i, k in enumerate(self.inventory):
+					if k == item:
```

Comment:
Let's make these variables more descriptive. That way another coder can understand what this function is working with quickly.